src/invert/fft_fftw.cxx

```diff
@@ -61,13 +61,47 @@ void fft_init(Options* options) {
   if (options == nullptr) {
     options = Options::getRoot()->getSection("fft");
   }
-  fft_init((*options)["fft_measure"]
-               .doc("Perform speed measurements to optimise settings?")
-               .withDefault(false));
+  bool fft_measure = (*options)["fft_measure"]
+                    .doc("Perform speed measurements to optimise settings?")
+                    .withDefault(false);
+  fft_flag = (*options)["fft_flag"]
+                    .doc("Level speed measurements to optimise FFT settings: [estimate], measure, exhaustive")
+                    .withDefault(FFT_FLAG::estimate);
+
+  if ((*options)["fft_measure"].isSet()) {
+    if ((*options)["fft_flag"].isSet()) {
+      throw BoutException("Cannot set both fft_measure and fft_flag");
+    }
+    fft_init(fft_measure);
+  } else {
+    fft_init(fft_flag);
+  }
+}
+
+unsigned int get_flags(FFT_FLAG fft_flag) {
```

Comment:
Currently this only deals with measurement flags but I guess we might want other options in the future as well like use wisdom or not so this may benefit from a more specific name (although may this is just something that could be done if we ever get to that situation)?

Reply:
Good point, maybe something like `fft_measurement_flag` would be better.

Reply:
+1 for `fft_measurement_flag`